blocks/editable/test/index.js

```diff
@@ -150,4 +150,47 @@ describe( 'Editable', () => {
 			} );
 		} );
 	} );
+
+	describe( 'Editable.Value', () => {
+		const Component = ( { value } ) => (
+			<div>
```

Comment:
We no longer need to wrap it with `div` in this test. This should make tests pass.